### PR TITLE
Update scriptfarewells

### PR DESCRIPTION
**Pull request by Ulvsblakk**
I implemented a date/time functionality for time specific farewells IE "Goodnight" etc. I don't know if this will interfere with other files using date/time as well. I also added a non time specific farewell as well. If there are any grammatical, or syntax errors, I will gladly revise.

### CODE REVIEW

File: Monika After Story/game/script-farewells.rpy [hunk 1]
```diff
@@ -44,3 +44,62 @@ label bye_untilwemeetagain:
     m 2e "'{i}Goodbyes are not forever, Goodbyes are not the end. They simply mean I’ll miss you, Until we meet again.{/i}'"
     m "Ehehe, 'till then, [player]!"
     return 'quit'
+    
+init 5 python:
+    addEvent(Event(persistent.farewell_database,eventlabel="i'll_see_you",random=True),eventdb=evhand.farewell_database)
+    
+label i'll_see_you:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-farewells.rpy]: Apostrophes cannot be part of labels. Also please prefix all farewell labels with `bye_`

**Comment by ThePotatoGuy** [comment]
Also, there's going to be a minor overhaul of the farewell system that should be able to account for time-variant greetings in a neater fashion, so we're not going to merge this until that is complete.

**Comment by Ulvsblakk** [comment]
I fixed everything. It should be good to go now.

**Comment by ThePotatoGuy** [comment]
Farewells has been overhauled and a new menu system has been included.
Please review the new system #1289 and make the appropriate changes to this pull request.

**Comment by Ulvsblakk** [comment]
Thanks for the update. I'll work on it when I get home from work.

File: Monika After Story/game/script-farewells.rpy [hunk 2]
```diff
@@ -44,3 +44,62 @@ label bye_untilwemeetagain:
...
+init 5 python: ## Implementing Date/Time for added responses based on the time of day
+    addEvent(Event(persistent.farewell_database,eventlabel="bye_prompt_haveagoodday",random=True,unlocked=True,pool=True),eventdb=evhand.farewell_database)
+label bye_prompt_haveagoodday
+    $ current_time = datetime.datetime.now().time().hour
+    if current_time >= 6 and current_time <= 11:
+        m 1b "Have a good day today [player]."
+        m 1b "I hope you accomplish everything you had planned for today."
+        m 2b "Even if you don't, don't get too discouraged, okay?"
+        m 2k "After all, there's always tomorrow."
+        m 1n "Sorry, I'm kind of rambling.
```
Comment (aldoram5) [inline Monika After Story/game/script-farewells.rpy]: here's also a missing double quote at the end

**Comment by Ulvsblakk** [comment]
Oh snap! I get it now. I'll work on it tomorrow after work. Thanks for the reply.

**Comment by Ulvsblakk** [comment]
Just wanted to let you guys know, I'm not dead. Works been super hectic lately and I haven't had much time to work on any of my requests. I'll have a bit of spare time on Tuesday. I'll be sure to get everything done then.

File: Monika After Story/game/script-farewells.rpy [hunk 3]
```diff
@@ -311,3 +311,97 @@ label bye_prompt_sleep:
...
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_enjoyyourafternoon",
+            unlocked=True,
+            rules=rules
+        ),
+        eventdb=evhand.farewell_database
+    )
+    del rules
+label bye_enjoyyourafternoon
```
Comment (aldoram5) [inline Monika After Story/game/script-farewells.rpy]: missing `:` at the end

File: Monika After Story/game/script-farewells.rpy [hunk 4]
```diff
@@ -311,3 +311,97 @@ label bye_prompt_sleep:
...
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_goodevening",
+            unlocked=True,
+            rules=rules
+        ),
+        eventdb=evhand.farewell_database
+    )
+    del rules
+label bye_goodevening
```
Comment (aldoram5) [inline Monika After Story/game/script-farewells.rpy]: missing `:` at the end

File: Monika After Story/game/script-farewells.rpy [hunk 5]
```diff
@@ -311,3 +311,97 @@ label bye_prompt_sleep:
...
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_goodnight",
+            unlocked=True,
+            rules=rules
+        ),
+        eventdb=evhand.farewell_database
+    )
+    del rules
+label bye_goodnight
```
Comment (aldoram5) [inline Monika After Story/game/script-farewells.rpy]: missing `:` at the end

**Pull request merged**